src/groupByKey.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.groupByKey = groupByKey;
+function groupByKey(items, key) {
+    var nObj = {};
+    for (var i = 0; i < items.length; i++) {
```

Comment:
You can use `let` instead of `var` for the loop variable `i` to limit its scope to the block, statement, or expression where it's used.